Cargo.toml

```diff
@@ -493,7 +493,7 @@ sources-internal_logs = []
 sources-internal_metrics = []
 sources-journald = []
 sources-kafka = ["dep:rdkafka"]
-sources-kubernetes_logs = ["dep:file-source", "kubernetes", "transforms-merge", "transforms-regex_parser"]
+sources-kubernetes_logs = ["dep:file-source", "kubernetes"]
```

Comment:
Possibly? Most of the deprecated transforms are a result of VRL/`remap` being able to replace them, which given that a lot of this stuff boils down to a facade over a Rust dependency, would lead to believe that we've simply shifted where the dependencies get used, rather than not actually needing them anymore.